app/models/solr_document.rb

```diff
@@ -16,10 +16,4 @@ class SolrDocument
   # and Blacklight::Document::SemanticFields#to_semantic_values
   # Recommendation: Use field names from Dublin Core
   use_extension(Blacklight::Document::DublinCore)
-  field_semantics.merge!(
-                         title: "title_tesi",
-                         author: "author_tesim",
-                         language: "language_ssim",
-                         format: "format_ssim"
-                       )
 end
```

Comment:
I am not sure why I removed this? @mkorcy you helped with a solo issue. was this you?

Reply:
I think I regenerated this file at some point. Adding this code back